charts/clusterapi-resources/values.yaml

```diff
@@ -187,3 +191,22 @@ bigipCtrlAS3:
     #   - 10.1.1.16
     #   - 10.1.1.17
     #   - 10.1.1.18
+
+# Configuration to make a cluster join an ArgoCD deployment
+argo:
+  enabled: false
+  namespace: argo
+  serviceAccount: # ServiceAccount to be created for the Pod - it will be cluster-admin
+    create: true
+    name: argo-manager
+  url: example.argocd.com # don't add the protocol
+  auth:
+    token: ""
+  version: 2.13.3
+  labels: {}
+    # labels that will be added to the Kubernetes Clusters managed by ArgoCD
+
+# basic setup to install a CNI to the cluster
+# this setup will use Helm to install
```

Comment:
This isn't quite true since we are using the CRS, even though its content was generated with a helm template I wouldn't say `use helm to install`, maybe based on a helm template or something along the lines.

Reply:
Updated the comments.